### PR TITLE
[Chrome] Keyboard Map and Keyboard Lock

### PR DESCRIPTION
**Pull request by jpmedley**
https://www.chromestatus.com/features/6730004075380736
https://www.chromestatus.com/features/5642959835889664

### CODE REVIEW

File: api/KeyboardLayoutMap.json [hunk 1]
```diff
@@ -0,0 +1,259 @@
+{
+  "api": {
+    "Lock": {
```
Comment (Elchi3) [inline api/KeyboardLayoutMap.json]: Lock -> KeyboardLayoutMap

File: api/KeyboardLayoutMap.json [hunk 2]
```diff
@@ -0,0 +1,259 @@
...
+            }
+          },
+          "status": {
+            "experimental": true,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "value": {
```
Comment (Elchi3) [inline api/KeyboardLayoutMap.json]: value -> values

File: api/KeyboardLayoutMap.json [hunk 3]
```diff
@@ -0,0 +1,259 @@
+{
```
Comment (Elchi3) [inline api/KeyboardLayoutMap.json]: KeyboardLayoutMap is defined here: https://wicg.github.io/keyboard-map/#keyboardlayoutmap-interface and it has "maplike" members (https://heycam.github.io/webidl/#idl-maplike).
So, you've added `entries`, `keys`, `size`, `values`. I think we also need `forEach`, `get`, and `has` then.

Comment (jpmedley) [reply api/KeyboardLayoutMap.json]: Not sure how I missed get and has. It doesn't look like forEach is part of maplike: https://heycam.github.io/webidl/#es-maplike

**Comment by jpmedley** [comment]
PTAL

File: api/KeyboardLayoutMap.json (hunk 3 above)
Comment (Elchi3) [reply api/KeyboardLayoutMap.json]: I would expect maplikes to have that method and if I inspect the object in Chrome Canary, there is `forEach`:

keybboardlayoutmap

**Comment by jpmedley** [comment]
I've corrected the issue, but I want one of Chrome engs to review it first.

File: api/KeyboardLayoutMap.json [hunk 4]
```diff
@@ -0,0 +1,361 @@
...
+            "deprecated": false
+          }
+        }
+      },
+      "size": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/KeyboardLayoutMap/size",
+          "support": {
+            "chrome": {
+              "version_added": "68"
```
Comment (garykac) [inline api/KeyboardLayoutMap.json]: KeyboardLayoutMap size should be M69

File: api/Navigator.json [hunk 5]
```diff
@@ -992,6 +992,57 @@
           }
         }
       },
+      "keyboard": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Navigator/keyboard",
+          "support": {
+            "chrome": {
+              "version_added": "69"
```
Comment (garykac) [inline api/Navigator.json]: navigator.keyboard was first addedin M68 (with Keyboard Lock)

**Comment by jpmedley** [comment]
@garykac PTAL

**Comment by jpmedley** [comment]
I remember doing it yesterday. I just don't know what happened to it.

**Pull request merged**